src/components/views/dialogs/MessageEditHistoryDialog.js

```diff
@@ -82,7 +90,23 @@ export default class MessageEditHistoryDialog extends React.PureComponent {
     render() {
         let content;
         if (this.state.error) {
-            content = this.state.error;
+            const {error} = this.state;
+            if (error.errcode === "M_UNRECOGNIZED") {
+                content = (<p className="mx_MessageEditHistoryDialog_error">
+                    {_t("Your homeserver doesn't seem to support this feature.")}
+                </p>);
+            } else if (error.errcode) {
+                // some kind of error from the homeserver
+                content = (<p className="mx_MessageEditHistoryDialog_error">
+                    {_t("Something went wrong!")}
+                </p>);
+            } else {
+                content = (<p className="mx_MessageEditHistoryDialog_error">
+                    {_t("Could not reach your homeserver to fetch the edit history.")}
+                    <br />
+                    {_t("Verify your internet connection or try again later.")}
```

Comment:
Maybe reuse the existing string "Ensure you have a stable internet connection, or get in touch with the server admin"? I guess it's missing a period... 😓